ipatool

```diff
@@ -1267,11 +1268,20 @@ def pr_list_command(ctx):
                 continue
             if lnames & labels_neg:
                 continue
+        commit_statuses = []
+        for c in pr.commits():
```

Comment:
I think we should only be interested in the "tip" commit.  If the previous tip was checked by CI, and later followed by a new commit as a fast-forward push, we should not include the CI status from the earlier commit(s).

Reply:
@abbra BTW, for how to find the tip commit see https://github.com/freeipa/freeipa-pr-ci/pull/349/files